### PR TITLE
Feat Category

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
1. Added UI for the Video Card

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Everything looks good. The code looks well formatted.

File: src/components/VideoListing.js [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+              <div className="video-title">
+                Highlight - Raptor Fortnite
+                <div className="video-subtitle">Fortnite meat</div>
+              </div>
+              <div className="video-extra-info">
+                <BiDotsVerticalRounded size={"1.2rem"} color="grey" />
+              </div>
+            </div>
+            <div className="video-subtitle sm-top-margin">
+              6K views &nbsp; | &nbsp; 4 hours ago
```
Comment (t007rushi) [inline src/components/VideoListing.js]: use icons instead

**Review by t007rushi — APPROVED** [review]
Good to merge with minor suggestion

File: src/components/VideoListing.js (hunk 1 above)
Comment (SimranjitKaurMaan) [reply src/components/VideoListing.js]: thanks for pointing out. Added icon.

**Pull request merged**